agent.go

```diff
@@ -65,14 +53,14 @@ type Agent struct {
 
 	Tags map[string]string
 
-	outputs []*runningOutput
-	plugins []*runningPlugin
+	Config *config.Config
 }
 
 // NewAgent returns an Agent struct based off the given Config
-func NewAgent(config *Config) (*Agent, error) {
+func NewAgent(config *config.Config) (*Agent, error) {
 	agent := &Agent{
 		Tags:          make(map[string]string),
```

Comment:
The Tags have been shuffled around a bit over time. There currently seems to be Tags attributes on both the Agent and the Config. I think the one on the Config is the correct one, so removing the one from the Agent may reveal a few places where it's still being used.